content/pages/home.yaml

```diff
@@ -8,4 +8,9 @@ description: A basic Grow demo website.
 ng_controller: BaseController
 
 components:
-- component: nav
+  - component: nav
+    logo: https://via.placeholder.com/450x150
```

Comment:
So you can nest objects in yaml.
So this could look like:
logo:
  url: http...
  alt: Alt test